### PR TITLE
Drop 'attempt to load non-existing icon' log for actions without icons

### PR DESCRIPTION
**Pull request by ahadas**
Add NoIcon annotation to explicitly state that no icon is set for a particular action, and in case the annotation is set for an action, we don't create a warning of 'attempt to load non-existing icon' for that action.

### CODE REVIEW

File: mucommander-core/src/main/java/com/mucommander/ui/action/NoIcon.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.mucommander.ui.action;
```
Comment (pskowronek) [inline mucommander-core/src/main/java/com/mucommander/ui/action/NoIcon.java:1]: copyright note?

File: mucommander-core/src/main/java/com/mucommander/ui/action/NoIcon.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```
Comment (pskowronek) [inline mucommander-core/src/main/java/com/mucommander/ui/action/NoIcon.java:6]: a javadoc explaining what is the idea behind?

**Comment by ahadas** [comment]
@pskowronek sorry, I missed your comments but I posted #1242 to address them

**Comment by pskowronek** [comment]
> @pskowronek sorry, I missed your comments but I posted #1242 to address them

np, I added them after the merge. Thanks.

**Pull request merged**